styleguide/components/31-molecules/building-blocks/quote/_quote.scss

```diff
@@ -0,0 +1,115 @@
+.quote {
+  @include theme('border-color', 'color-primary--lighten-4', 'quote-border-color');
+
+  @supports (display: grid) {
```

Comment:
What if it's not supported?

Reply:
Then it gets ignored :)

Reply:
Screen shots would be nice. :wink: 
If the result is the same, why use it at all?

Reply:
Because IE11 does not support it completely... using this we make sure that it uses css grid properties only when the browser supports the latest spec...
If we don't use this some properties might be used while other won't be and we get unexpected results in IE browsers...

Reply:
I know what @supports does :stuck_out_tongue_winking_eye: 
Will drop by this afternoon, this seems overly complicated for what you are trying to achieve.